xx.md

```diff
@@ -0,0 +1,63 @@
+NIP-XX
+=======
+
+Invite Links
+------------
+
+In order to facilitate exchange of nostr recommendations, it's useful to have a single link with multiple
+entities encoded in it. This can be especially helpful as part of an onboarding process, to get new users
+started with recommendations hand-picked by another user.
+
+This NIP defines a new bech32-encoded entity, `ninvite1` and a new event representing an invite code which
+will grant access to a given relay.
+
```

Comment:
I would get rid of the two initial paragraphs and simplify them to: 

This NIP defines a new bech32-encoded entity, `ninvite1`, to encode recommendations for existing and new nostr users.